docs/_data/portland-2019-day-1.yaml

```
@@ -68,16 +68,9 @@
     \ yourself and your users into in-app messaging."
   Slug: meghan-mahar
   Time: '4:10'
-- Duration: '10'
-  Session: Switch Speakers
-  Time: '4:40'
-- Duration: '30'
-  Session: Mark F Iverson - Harvest Past Experience to be a Great Tech Writer
-  Slug: mark-f-iverson
-  Time: '4:50'
 - Duration: '10'
   Session: Group Photo
-  Time: '5:20'
+  Time: '4:40'
 - Duration: '0'
   Session: Day 1 wraps up
-  Time: '5:30'
+  Time: '4:50'
```

```suggestion
  Time: '5:00'
```

Just to make it round out nicely.